jenkins/upgrade_python_requirements.py

```diff
@@ -0,0 +1,76 @@
+"""
+Script to help create a PR with python library upgrades. To be run inside
+a Jenkins job that first runs `Make upgrade`
+"""
+import logging
+
+import click
+
+from github_helpers import (authenticate_with_github, branch_exists,
+                            connect_to_repo, create_branch, close_existing_pull_requests,
+                            create_pull_request, delete_branch,
+                            get_modified_files_list, update_list_of_files)
+
+logging.basicConfig()
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+@click.command()
+@click.option(
+    '--sha',
+    help="Sha of the merge commit to base the new PR off of",
+    required=True,
+)
+@click.option(
+    '--repo_root',
+    help="Path to local repository that will hold updated database files. "
+    "Make sure the path includes the repo name",
+    required=True,
+)
+def main(sha, repo_root):
+    logger.info("Authenticating with Github")
+    github_instance = authenticate_with_github()
+
+    # Last folder in repo_root should be the repository
+    directory_list = repo_root.split("/")
+    repository_name = directory_list[-1]
+    logger.info("Trying to connect to repo: {}".format(repository_name))
+    repository = connect_to_repo(github_instance, repository_name)
+
+    modified_files = get_modified_files_list(repo_root)
+    if len(modified_files) > 0:
+        branch = "refs/heads/jenkins/upgrade-python-requirements-{}".format(sha[:7])
+        if branch_exists(repository, branch):
+            logger.info("Branch for this sha already exists")
+        else:
+            git_tree = repository.get_git_tree(sha)
+            modified_files_list = modified_files.split("\n")
```

Comment:
Based on the helper function name `get_modified_files_list`, I'd expect it to have already done this and returned a Python list.